flexget/components/notify/notifiers/gotify.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import base64
+import datetime
+import re
+
+from flexget import plugin
+from flexget.event import event
+from flexget.config_schema import one_or_more
+from flexget.plugin import PluginWarning
+from flexget.utils.requests import Session as RequestSession, TimedLimiter
+from requests.exceptions import RequestException
+from http import HTTPStatus
+from urllib.parse import urljoin
```

Comment:
please sort imports correctly. stdlib, 3rd party and then flexget.
if you use and ide like pycharm you can do it via hotkey

Reply:
https://github.com/Flexget/Flexget/pull/2533/commits/f5a6ef872f61b3a848de12a98ea095432091fad9